src/v2/Apps/Fair/Components/FairOverview/FairAbout.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useMemo } from "react"
+import { Column, GridColumns, Separator, Spacer, Text } from "@artsy/palette"
+import { moment } from "desktop/lib/template_modules"
+import { some } from "lodash"
+import { InfoSection } from "v2/Components/InfoSection"
+import { FairTimerFragmentContainer as FairTimer } from "./FairTimer"
+import { createFragmentContainer, graphql } from "react-relay"
+import { FairAbout_fair } from "v2/__generated__/FairAbout_fair.graphql"
+
+const aboutInfoTypes = [
+  "about",
+  "tagline",
+  "location.summary",
+  "ticketsLink",
+  "hours",
+  "links",
+  "contact",
+  "summary",
+  "tickets",
+]
+
+interface FairAboutProps {
+  fair: FairAbout_fair
+}
+
+const FairAbout: React.FC<FairAboutProps> = ({ fair }) => {
+  const {
+    about,
+    tagline,
+    location,
+    ticketsLink,
+    hours,
+    links,
+    contact,
+    summary,
+    tickets,
+    endAt,
+  } = fair
+
+  const hasEnded = endAt && moment().isAfter(new Date(endAt))
```

Comment:
**suggestion**: Can we take this opportunity to drop the moment import here?

As mentioned [here](https://github.com/artsy/force/pull/8169#discussion_r687913313), we're trying to move away from moment entirely. If we still need this `hasEnded` conditional, there should be a similar solution available using luxon.

But I'm wondering, even if the fair hasn't ended, should we still hide the about section if there's no content to display? I forget why we have this `hasEnded` conditional to begin with.

Reply:
If a fair has ended we shouldn't display timer. I'm not sure if we should hide empty About section in this case

Reply:
Right, hiding the timer makes sense although I think we can continue to show the about content if it's available.

Reply:
You are right. And we dislpay about info if it`s available. In the component we return null only if fair has ended and there are not anything info about it
It checked in 47-49 lines in FairAbout component